zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -1088,7 +1088,10 @@ private InterpreterSetting getDefaultInterpreterSetting(List<InterpreterSetting>
   public InterpreterSetting getDefaultInterpreterSetting(String noteId) {
     return getDefaultInterpreterSetting(getInterpreterSettings(noteId));
   }
-
+  
+  public boolean isBinding(String noteId, String replName) {
```

Comment:
Would `isBinded` make more sense as a name for a method that checks weather a particular note is binded to a particular interpreter?